components/interactive_dialog/dialog_element/dialog_element.tsx

```diff
@@ -7,7 +7,7 @@ import {FormattedMessage} from 'react-intl';
 
```

Comment:
You need to change the type definition of `autocompleteChannels` 
- from `autocompleteChannels: (term: string, success: (channels: Channel[]) => void, error: () => void) => Promise<void>;`
- to `autocompleteChannels: (term: string, success: (channels: Channel[]) => void, error?: (err: ServerError) => void) => (ActionResult | Promise<ActionResult | ActionResult[]>);`